Makefile

```diff
@@ -52,47 +52,40 @@ clean-test: ## remove test and coverage artifacts
 lint: ## check style with flake8
 	pipenv run flake8 --ignore=E501 git_wrapper tests integration_tests
 
-test: dev ## run tests quickly with the default Python
-	pipenv run py.test tests
+test: ## run tests quickly with the default Python
+	pipenv run pytest
 
-test-all: dev ## run tests on every Python version with tox
+test-all: ## run tests on every Python version with tox
 	pipenv run tox
 
 coverage: ## check code coverage quickly with the default Python
-	pipenv run coverage run --source git_wrapper -m pytest
-	pipenv run coverage report -m
-	pipenv run coverage html
-	$(BROWSER) htmlcov/index.html
+	pipenv run pytest --cov-report=term-missing --cov=git_wrapper tests
+	## $(BROWSER) htmlcov/index.html
 
 docs: ## generate Sphinx HTML documentation, including API docs
-	rm -f docs/source/git_wrapper.rst
-	rm -f docs/source/modules.rst
-	sphinx-apidoc -o docs/source git_wrapper
-	$(MAKE) -C docs clean
-	$(MAKE) -C docs html
+	pipenv run $(MAKE) -C docs clean
+	pipenv run $(MAKE) -C docs html
 	$(BROWSER) docs/_build/html/index.html
 
 servedocs: docs ## compile the docs watching for changes
 	pipenv run watchmedo shell-command -p '*.rst' -c '$(MAKE) -C docs html' -R -D .
 
 release: clean ## package and upload a release
-	pipenv run python setup.py sdist upload
-	pipenv run python setup.py bdist_wheel upload
+	pipenv run twine check dist/*
```

Comment:
We seem to check --strict in the tox job used in the gh action, should we do that here too to stay consistent and avoid surprises? I wonder if we should call the Make commands where possible in the gh actions too if we're keeping the Makefile, to avoid more duplication.

Reply:
Great idea, I'll change this to call tox -etwine like the github action.